Yafc.UI/ImGui/ImGuiTextInputHelper.cs

```diff
@@ -271,27 +303,34 @@ public bool KeyDown(SDL.SDL_Keysym key) {
                 }
 
                 break;
```

Comment:
again a lot of breaks with a newline above and below. But hereafter here the style breaks and goes back to single newline.

Newlines to me tell a story, or rather, break up a story. Like an English text has paragraphs, code has too. Putting newline after certain structures breaks that. If it's a big block than adding an extra newline after a `break`? Sure. But a single block with just `break` doesn't tell me an interesting story.

Reply:
The blank line above the break is due to a code-block ending. I'll take a look at breaks again tomorrow with a fresh look, to see if I overdid blank lines.

Reply:
Looks a lot better